content/rest-api/filter.md

```diff
@@ -274,22 +274,16 @@ In the above url :
 
 #### Examples
 
-To only retrieve product models of the `winter_collection` category, you can use the following URL.
+To filter product models on their identifier, use the `identifier` property.
+Here are the allowed operators you can use to filter on the identifier as well as the corresponding type of value expected in the `search` query parameter.
 
-```
-/api/rest/v1/product-models?search={"categories":[{"operator":"IN","value":["winter_collection"]}]}
-```
-
-Of course, you can combine as many filters as you want. The example below will get you the product models being 100% complete on the en_US locale.
-
-```
-/api/rest/v1/product-models?search={"categories":[{"operator":"IN","value":["winter_collection"]}],"completeness":[{"operator":"ALL COMPLETE","locale":"en_US","scope":"ecommerce"}]}
-```
+| Allowed operators | Allowed value type           |
+| ----------------- |------------------------------|
+| IN, NOT IN        | list of strings (identifier) |
```

Comment:
do you really want to remove the other example?

Reply:
the examples that I removed were duplicated below in the chapters "On categories" and "On completeness"